app/src/main/java/org/simple/clinic/settings/SettingsEffectHandler.kt

```diff
@@ -17,12 +18,16 @@ class SettingsEffectHandler @AssistedInject constructor(
     private val schedulersProvider: SchedulersProvider,
     private val appVersionFetcher: AppVersionFetcher,
     private val appUpdateAvailability: CheckAppUpdateAvailability,
-    @Assisted private val uiActions: UiActions
+    @Assisted private val uiActions: UiActions,
+    @Assisted val viewEffectsConsumer: Consumer<SettingsViewEffect>
```

Comment:
Is there a reason why `viewEffectsConsumer` is not private?

Reply:
Because I missed it 😂. Good catch.